packages/gguf/src/transformer-llm.ts

```diff
@@ -62,10 +62,10 @@ export const LLM_ARCHITECTURES = [
 	"mpt",
 	"baichuan",
 	"starcoder",
-	"persimmon",
```

Comment:
they dropped this arch?

Reply:
Yes it's removed in https://github.com/ggerganov/llama.cpp/pull/7408

Btw, should I merge it now, or wait for approval from @mishig25 ? Thanks.

Reply:
@mishig25 is OOO so you can merge 🔥